exec/java-exec/src/test/java/org/apache/drill/exec/impersonation/TestInboundImpersonation.java

```diff
@@ -156,22 +159,25 @@ public void unauthorizedTarget() throws Exception {
 
   @Test
   public void invalidPolicy() throws Exception {
-    thrownException.expect(new UserExceptionMatcher(UserBitShared.DrillPBError.ErrorType.VALIDATION,
-        "Invalid impersonation policies."));
+    String query = "ALTER SYSTEM SET `%s`='%s'";
```

Comment:
We have methods that do this work: it is very common. Maybe the required clients are not available in this particular test?

Reply:
Hi Paul,
Greetings to you!
I did this because the UT code in Drill used `ExpectedException thrown = ExpectedException.none();`
for ExpectedException instantiation.
However the method `ExpectedException.none()` has been deprecated, so I replaced `ExpectedException.none()` by `Assert.assertThrows` as the [deprecation notice](https://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html#none()) suggested.
Could you please give some suggestions will be really appreciated.
Best,
Jingchuan

Reply:
Just picking one test file at random, here's an example of the idea: https://github.com/apache/drill/blob/master/exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestLargeFileCompilation.java#L273

It is not about the exception handling, which is fine. It's just about the busy-work of formatting the `ALTER SYSTEM SET` commands over and over.

Again, without looking closely, I don't know if this test happens to use one of the two clients that have these methods. If not, then you have to do things the tedious way.

Reply:
Hi Paul, it does really help! I will replace all those tedious work by client in this PR, thanks!

Reply:
@paul-rogers Hi Paul, I am sorry that I dragged this PR out too long. I replaced hard-code to FixtureClient, but UT failed due to the Error:  
`java.lang.AssertionError: unexpected exception type thrown; expected:<org.apache.drill.common.exceptions.UserException> but was:<java.lang.IllegalStateException>
	at org.apache.drill.exec.impersonation.TestInboundImpersonation.invalidProxy(TestInboundImpersonation.java:186)
Caused by: java.lang.IllegalStateException: 
org.apache.drill.common.exceptions.UserRemoteException: PERMISSION ERROR: Cannot change option exec.impersonation.inbound_policies in scope SESSION`  
I think the two clients cannot exist at the same time, so I changed back to the hard-code way.